src/random.cpp

```diff
@@ -436,7 +465,23 @@ static void SeedSlow(CSHA512& hasher) noexcept
     SeedTimestamp(hasher);
 }
 
-static void SeedSleep(CSHA512& hasher)
+/** Extract entropy from rng, strengthen it, and feed it into hasher. */
+static void SeedStrengthen(CSHA512& hasher, RNGState& rng) noexcept
+{
+    // Once per minute, strengthen the hash for 10ms.
+    static std::atomic<int64_t> last_strengthen;
+    int64_t current_time = GetTimeMicros();
+    if (current_time > last_strengthen + 60000) {
+        // Generate 32 bytes of entropy from the RNG.
+        unsigned char buf[32];
+        rng.MixExtract(buf, 32, CSHA512(), false);
```

Comment:
Nit: might be slightly more readable to rename `buf` to `strengthener_seed`, or expand the comment with ", to seed the strengthener."